executor/show_stats_test.go

```diff
@@ -203,3 +206,29 @@ func (s *testShowStatsSuite) TestShowAnalyzeStatus(c *C) {
 	c.Assert(result.Rows()[1][5], NotNil)
 	c.Assert(result.Rows()[1][6], Equals, "finished")
 }
+
+func (s *testShowStatsSuite) TestShowStatusSnapshot(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("drop database if exists test;")
+	tk.MustExec("create database test;")
+	tk.MustExec("use test;")
+	tk.MustExec("drop table if exists t;")
```

Comment:
We create a new database here, it seems that we can remove this line.

Reply:
done